dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -94,6 +94,7 @@ namespace dwa_local_planner {
       limits.acc_lim_theta = config.acc_lim_theta;
       limits.acc_limit_trans = config.acc_limit_trans;
       limits.acc_limit_tip = config.acc_limit_tip;
+      ROS_INFO("Config xy tolerance: %f", config.xy_goal_tolerance);
```

Comment:
We should probably remove these since the dynamic reconfigure values are printed at startup and can be checked in rqt.

Reply:
Yeah...

Reply:
done